src/OrchardCore.Modules/OrchardCore.Liquid/README.md

```diff
@@ -21,7 +21,7 @@ Returns the url of the content item
 
 Input
 ```
-{{ ContentItem | display_url }}
```

Comment:
Did you check with a Content Type template, like `Content__Blog` ?

Reply:
Just tried with the `Content__Article` because i'm on the agency theme.

Reply:
This is because at one point we provided this property at the top level of the context. But to be consistent with razor view, we now use `Model.ContentItem`.